fahadmak/Nai-Java-Devs
# #156187450 Create Network utility class

#### What does this PR do?
- [x] This PR  creates a network utility class to detect network connectivity for the application views.

#### Description of Task to be completed?
- [x] Create a NetworkUtility class
- [x] Create NetworkListener interface

#### How should this be manually tested?
- Clone the repo and cd into it
- Checkout to origin/ft-UI/UX-Revamp-15603724
- Open the project in Android Studio
- Run the android emulator
- Turn on and off the internet to see the snackbar

#### What are the relevant pivotal tracker stories?
[#156187450](https://www.pivotaltracker.com/story/show/156187450)

#### Screenshots (if appropriate)
<img width="200" alt="Screen Shot 2019-03-25 at 6 28 43 PM" src="https://user-images.githubusercontent.com/13882936/56129737-92d47200-5f8b-11e9-8b72-0098a5c8c997.gif">
